src/module.c

```diff
@@ -46,18 +49,50 @@ static int _Setup_ThreadPOOL(int threadCount) {
 	return 1;
 }
 
+// Sets the golbal variable of the redis major version
+static void _SetRedisMajorVersion(RedisModuleCtx *ctx) {
+
+	const char *server_version;
+	int major = 5;
+	int minor;
+	int minor_minor;
+	// Check if there is an implementation for redis module api for redis 6 an up.
```

Comment:
This comment should lengthily explain that you're null-checking a function pointer that will be null on Redis 5.